### ISSUE TITLE
Question about blurhash / dominantColor

### ISSUE THREAD

**Issue by rw-ninety** [spacecatninja/craft-imager-x-power-pack#1]
Am I correct these don't work when using Imageboss? The blurhash is transparent and the dominantColor outputs: background-color: ; (empy) in the style tag. 

Thanks.

**Comment by aelvan** [comment]
Yes, ImageBoss doesn't support getting that information, so the transformed image model doesn't return it. But, powerpack should maybe just fall back to using a craft transformer for these things. Maybe behind a config setting. I'll give it some thought and see what I decide on.

**Comment by rw-ninety** [comment]
Hi André, that's a great suggestion and was actually something I was looking for in the docs.

**Comment by aelvan** [comment]
This has been resolved in 1.0.1. 

There was a small inconsistency, where the `ppplaceholder` functionality relied on the local `craft` transformer, but the one enabled by the configuration didn't. I just made both base the placeholder on a local transform, since it's the only way to get a consistent behaviour across different transformers. This might not be ideal in cases where users absolutely don't want, or can have, transforms locally, but then the placeholder functionality can be solved outside of the context of the Power Pack.
